packages/downloading-helpers/src/scripts/replay-assets.ts

```diff
@@ -66,6 +74,15 @@ export const fetchAsset: (path: string) => Promise<string> = async (path) => {
   }
 };
 
+async function fetchSourceMapIfDebugging(path: string) {
+  if (
+    process.env.METICULOUS_SNIPPETS_BASE_URL?.includes("localhost") &&
```

Comment:
[nit] Could use `getSnippetsBaseUrl()` if want to keep the magic constant name in one place:

```suggestion
    getSnippetsBaseUrl().includes("localhost") &&
```

Reply:
Aha, yeah that's better. Done.